dagql_parser.py

```diff
@@ -24,15 +24,15 @@ def error(self,token_type=None):
         raise Exception("Expected %s Token in line %s, got %s" % \
             (token_type, self.current_token.line, self.current_token.type))
     def eat(self, token_type):
-        print(self.current_token)
+        # print(self.current_token)
```

Comment:
Instead of commenting them out, can you delete these lines instead?

Reply:
I am just not sure if @CoffeeTableEspresso still needs it. that's why I comment it out instead of deleting it entirely. @CoffeeTableEspresso ?

Reply:
The whole point of version control is that everything is in history. So if something is not needed, it should be deleted.